src/index.test.js

```diff
@@ -31,6 +31,20 @@ describe('<img> mode', () => {
     expect(vdom.props.src).toInclude(src)
   })
 })
+describe('<img> mode - pre-mount', () => {
+  beforeEach(() => {
+    tree = sd.shallowRender(
+      <Imgix
+        src={src}
+      />
+    )
+    vdom = tree.getRenderOutput()
+  })
```

Comment:
Is this emulating the pre-mount state because `tree.getMountedInstance()` is not present?